lib/rucio/rse/rsemanager.py

```diff
@@ -68,31 +68,34 @@ def get_rse_info(rse, session=None):
 def _get_possible_protocols(rse_settings, operation, scheme=None):
     operation = operation.lower()
     candidates = rse_settings['protocols']
+
     if type(rse_settings['domain']) is not list:
         raise exception.RSEProtocolDomainNotSupported('Domain setting must be list.')
 
-    for d in rse_settings['domain']:
-        if d not in utils.rse_supported_protocol_domains():
-            raise exception.RSEProtocolDomainNotSupported('Domain %s is not supported by Rucio.' % d)
-
     # convert scheme to list, if given as string
     if scheme and not isinstance(scheme, list):
         scheme = scheme.split(',')
 
     tbr = []
```

Comment:
Maybe declaring tbr as set could do the trick? Or just converting tbr to a set before using len below

Reply:
TypeError: unhashable type: 'dict'

Reply:
Ok sorry. I was told protocol would be a string

Reply:
scheme yes but no worry the code there is ...... interesting !